src/pipelines/zero_shot_classification.rs

```diff
@@ -706,22 +706,30 @@ impl ZeroShotClassificationModel {
     ///         sentence: 1,
     ///     },
     /// ]
-    /// .to_vec();
+    /// .to_vec());
     /// ```
-    pub fn predict<'a, S, T>(
+    pub fn try_predict<'a, S, T>(
         &self,
         inputs: S,
         labels: T,
         template: Option<ZeroShotTemplate>,
         max_length: usize,
-    ) -> Vec<Label>
+    ) -> Result<Vec<Label>, RustBertError>
     where
         S: AsRef<[&'a str]>,
         T: AsRef<[&'a str]>,
     {
         let num_inputs = inputs.as_ref().len();
         let (input_tensor, mask) =
-            self.prepare_for_model(inputs.as_ref(), labels.as_ref(), template, max_length);
+            match self.prepare_for_model(inputs.as_ref(), labels.as_ref(), template, max_length) {
+                Some((input_tensor, mask)) => (input_tensor, mask),
+                None => {
+                    return Err(RustBertError::ValueError(
```

Comment:
Following my previous comment, I think it would be beneficial to raise the error closer to the problematic call to `max()` in the `prepare_for_model` (and just use `?` on its output)